tests/framework/helpers/ArrayHelperTest.php

```diff
@@ -1436,6 +1436,28 @@ public function testArrayAccessWithMagicProperty()
         $this->assertEquals(42, ArrayHelper::getValue($model, 'magic'));
         $this->assertEquals('ta-da', ArrayHelper::getValue($model, 'moreMagic'));
     }
+
+    public function testSortMultidimensionalArray()
```

Comment:
please, use data provider, see [Example 2.6 Using a data provider with named datasets](https://phpunit.readthedocs.io/en/7.5/writing-tests-for-phpunit.html#writing-tests-for-phpunit-data-providers-examples-datatest1-php) and add complex data to tests:
~~~php
['c' => ['bar' => 1, 'baz' => 1], 'a' => ['baz' => 2, 'bar' => 1], 'b' => '...'];

[['attr', 'attr2'], 'equal', 'option' => 'value', 'option2' => 'value2'];
[['attr2', 'attr'], 'equal', 'option2' => 'value2', 'option' => 'value'];
~~~

Reply:
@WinterSilence Please check the modifications for the new test cases